playground/frontend/playground_components/lib/enums.dart

```diff
@@ -0,0 +1 @@
+enum Complexity { basic, medium, advanced }
```

Comment:
No license here.
Please run `./gradlew rat` to spot all such files.

Since you will be editing this anyway, its better to list each constant on a separate line.

Reply:
Also it's better to have each `enum` in a separate file under `enums` directory. This pays when dealing with large merges later.